src/node_dotenv.cc

```diff
@@ -52,6 +52,11 @@ void Dotenv::SetEnvironment(node::Environment* env) {
     auto existing = env->env_vars()->Get(key.data());
 
     if (existing.IsNothing()) {
+      size_t found = value.find("\\\"");
```

Comment:
This needs to be done inside the ParseLine function. Depending on the start character of the value, we need to de-escape it. For ' it is \', or for " it is \"